### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by james-laing**
TLS doc update to include usage example and minor correction to grammar and links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,16 +187,16 @@ scrape_configs:
...
-  - targets:           #  <== the list of your flasharrays goes here
+  - targets:           #  <== the list of your FlashArrays goes here
     - 10.11.12.80
     - 10.11.12.82
     - 10.11.12.90
 
 ```
 
-See the kubernetes [examples](examples/config/k8s) for a similar configuration that uses a the additional config for a simple Prometheus kubernets deploymemt or the more interesting Prometheus operator.
+See the Kubernetes [examples](examples/config/k8s) for a similar configuration that uses a the additional config for a simple Prometheus Kubernets deployment or the more interesting Prometheus operator.
```
Comment (chrroberts-pure) [inline README.md:197]:
```suggestion
See the Kubernetes [examples](examples/config/k8s) for a similar configuration that uses additional configuration items for a simple Prometheus+Kubernetes deployment, or for the more interesting Prometheus operator.
```

File: README.md [hunk 2]
```diff
@@ -237,6 +237,23 @@ A simple but complete example to deploy a full monitoring stack on Kubernetes ca
 
 A complete example monitoring stack implemented in Docker Compose which can be found in the [examples](examples/config/docker) directory. 
 
+#### TLS HTTPS Support
+
+An usage example of how to deploy the exporter with TLS.
```
Comment (chrroberts-pure) [inline README.md:242]:
```suggestion
An usage example of how to deploy the exporter with TLS using the `pure-fa-om-exporter` binary.
```

File: README.md [hunk 3]
```diff
@@ -237,6 +237,23 @@ A simple but complete example to deploy a full monitoring stack on Kubernetes ca
...
+
+Deployment:
+```shell
+$ pure-fa-openmetrics-exporter/out/bin/pure-fa-om-exporter -c cert.crt -k cert.key
+2023/08/01 12:00:00 Start Pure FlashArray exporter v1.0.9 on 0.0.0.0:9490
+```
+Testing:
+```shell
+$ cURL --header 'Authorization: Bearer xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx' --request GET 'https://pure-fa-exporter.your.domain:9490/metrics/array?endpoint=arrayname.your.domain' --insecure --silent | grep purefa_info
```
Comment (chrroberts-pure) [inline README.md:251]: We need to standardize the variable we use for `<exporter-host>` . There's 3 different uses of that var in this document that use a different name

Comment (james-laing) [reply README.md:251]: I agree, I think an overhaul of this page is required now that it has grown. I propose we tackle these issues as part of a refresh and agree on the standardised names which is out of scope of this PR.

File: extra/grafana/README.md [hunk 4]
```diff
@@ -52,10 +52,12 @@ This deployment assumes the [Pure Storage FlashArray OpenMetrics Exporter][1] is
 Supported operating system platforms are available to install Prometheus and Grafana.
 
 The Grafana dashboards have been developed and tested using the following software versions:
```
[the remark this answers is not in the document]
Comment (james-laing) [reply extra/grafana/README.md:55]: Thanks - I agree. This is fixed in commit d18d268

File: README.md (hunk 2 above)
Comment (james-laing) [reply README.md]: Agreed, I have submitted a new commit to cover this and another minor adjustment in commit 2945c45.

**Pull request merged**